index.html

```diff
@@ -51,6 +51,20 @@ <h1 class="title-resturaunt">FoodCity</h1>
     <button class="btn"><a href="home.html">Go to Menu</a></button>
 </section>
 <!-- End Home Section -->
+
+<!-- Start About Us Section  -->
+    <section id="about-us">
```

Comment:
This section shouldn't appear without scrolling in tablet size page

Reply:
It's working in my browser, what the problem